ext/spl/spl_dllist.c

```diff
@@ -1257,6 +1257,10 @@ PHP_METHOD(SplDoublyLinkedList, add)
 
 		/* Get the element we want to insert before */
 		element = spl_ptr_llist_offset(intern->llist, index, intern->flags & SPL_DLLIST_IT_LIFO);
+		if (element == NULL) {
```

Comment:
I'm not sure that `element` can ever by `NULL` here. Doesn't look like that would be possible, in which case we could add a simple assertion instead:
````.c
ZEND_ASSERT(element != NULL);
````

Reply:
> I'm not sure that `element` can ever by `NULL` here. Doesn't look like that would be possible, in which case we could add a simple assertion instead:
> 
> ```c
> ZEND_ASSERT(element != NULL);
> ```

Thanks for your patience. I have already changed to zend_assert.